scripts/core/superdesk-api.d.ts

```diff
@@ -897,6 +898,9 @@ declare module 'superdesk-api' {
 
         field: string;
 
+        /** default value */
+        value?: string;
```

Comment:
Is this used anywhere?

Reply:
I had it for that hidden field, can remove it now